### ISSUE TITLE
Strange class with tamed entities

### ISSUE THREAD

**Issue by WyRe** [slipcor/pvparena#295]
I would like to create a class like Druid, with two tamed wolfs, but I've tried and, obviously wolfs are not items so the won't be loaded when people use the signal. Could be possible to add this feature in a future? What do you think?

**Comment by slipcor** [comment]
Yes, it should be possible - I have added this as a perk for the "Upgrade" module - I think I will try to add this this or next week, implementation will be a bone item with name "SPAWN" :)

**Comment by WyRe** [comment]
Thank you so much! :D
